libraries/stumbler/src/main/java/org/mozilla/mozstumbler/service/core/http/HttpUtil.java

```diff
@@ -102,7 +96,8 @@ public String getUrlAsString(String url) throws IOException {
 
     @Override
     public InputStream getUrlAsStream(String url) throws IOException {
-        return new URL(url).openStream();
```

Comment:
I had 2 different ways of handling URLs before - this has been consolidated so that we always use the mozOpenConnection function for URLs from now on.